draft-ietf-mls-protocol.md

```diff
@@ -1935,9 +1935,8 @@ The client verifies the validity of a LeafNode using the following steps:
     leaf node was proposed for addition, even if it is expired at these later
     points in the protocol.
 
-* Verify that the `leaf_node_source` field has the appropriate value for the
-  context in which the LeafNode is being validated (as defined in
-  {{leaf-node-contents}}).
+* If applicable, verify the `parent_hash` as specified in
+  {{verifying-parent-hashes}}.
```

Comment:
This doesn't make sense.  It's not meaningful to just verify the parent hash in a leaf.  For example, a leaf's parent hash might be invalid just because its sibling updated more recently.  Parent hashes should really only be validated (a) on validating a newly received tree, and (b) when validating a Commit UpdatePath.